src/types/state.rs

```diff
@@ -16,9 +21,16 @@ pub enum State {
 
     // Awaiting for https://github.com/rust-lang/rust/issues/44109
     #[doc(hidden)]
+    #[serde(skip)]
     __Nonexhaustive,
```

Comment:
You can just use `#[non_exhaustive]` now or perhaps just get rid of this—no need to avoid breaking changes in a vendored dependency.

Reply:
You mean that I can remove `__Nonexhaustive` entirely, right? Just want to make sure that I understood that correctly.

Reply:
Sure.